pkg/workflows/testdata/fixtures/workflows/marshalling/workflow_2_spec.json

```diff
@@ -1,33 +1,49 @@
 {
-  "name": "length_ten",
-  "owner": "0x0123456789abcdef0123456789abcdef01234567",
-  "triggers": [
+  "Name": "length_ten",
```

Comment:
why did the case change? is it important that it changes?

Reply:
i think this might be b/c of the struct change that removes the `json` tags (and `json schema`)

if that's true, then since we are not supporting json encoding of this struct in an meaningful way, i think we can delete any tests that use this file and delete it

Reply:
Yes, correct :) I've removed it from StepInput as well so we're consistent now.

Note that we do need these tests since they guarantee that we convert from YAML -> JSON consistently and in a lossless way.

Reply:
we DO NOT convert from yaml to json in a lossless way.

it is one of the features of intermediate YAML representation, specifically that versions can be represented in YAML multiple ways  that result in the identical JSON.

test: https://github.com/smartcontractkit/chainlink-common/blob/97d4285d9f4871aafd3a88833c6551c850929a40/pkg/workflows/models_yaml_test.go#L167
doc: https://github.com/smartcontractkit/chainlink-common/blob/97d4285d9f4871aafd3a88833c6551c850929a40/pkg/workflows/dependency_graph.go#L30

did i misunderstand what you meant or did you think the behavior is different?

Reply:
@krehermann No, you're right -- I was referring to things such as handling conversions properly; not the id being stringified regardless of whether it's a table or not